src/main/menu/menu.ts

```diff
@@ -59,19 +59,19 @@ const template = [
             {
                 label: "Discord Support",
                 click: async (): Promise<void> => {
-                    await shell.openExternal(process.env.DISCORD_URL);
+                    await shell.openExternal("https://discord.gg/4GBwH52cFb");
```

Comment:
Ain't this link suppose to be different like inviting to specific support channel?